checador/app/controllers/records_controller.rb

```diff
@@ -0,0 +1,4 @@
+class RecordsController < ApplicationController
+  def employees_records   
```

Comment:
This new method is going to fail because you are not adding the view, please ensure to add the view file in this same commit, and why you are not using the `new_employee_path` route as it was before?

Reply:
Hi!
I created this new method with that new controller because it is what we will use for employee check in and check out.
The old route was an example but that will be used to create the employees in the administrator part.